app/Presenters/Moderations/StickPresenter.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/**
+ * @author    MyBB Group
+ * @version   2.0.0
+ * @package   mybb/core
+ * @license   http://www.mybb.com/licenses/bsd3 BSD-3
+ */
+
+namespace MyBB\Core\Presenters\Moderations;
+
+use MyBB\Core\Moderation\Moderations\Stick;
+
+class StickPresenter extends AbstractReversibleModerationPresenter implements ReversibleModerationPresenterInterface
+{
+    /**
+     * @return Stick
+     */
+    public function getWrappedObject()
+    {
+        return parent::getWrappedObject();
+    }
+
+    /**
+     * @return string
+     */
+    public function icon()
+    {
+        return 'fa-sticky-note';
```

Comment:
@Eric-Jackson Just a note about our discussions regarding SVG icons - these FontAwesome classes are used in all the moderation things - we need to remember to change/fix this somehow!!! (pinging you so that you'll hopefully nudge me when the time comes 😉)